ops/model.py

```diff
@@ -34,6 +34,8 @@
 import warnings
 import weakref
 from abc import ABC, abstractmethod
+from contextlib import contextmanager
+from contextvars import ContextVar
```

Comment:
Our style is to import `contextlib` and `contextvars`.

Reply:
done.